source/slang/slang-capability.h

```diff
@@ -322,6 +322,9 @@ bool isCapabilityDerivedFrom(CapabilityAtom atom, CapabilityAtom base);
     /// Find a capability atom with the given `name`, or return CapabilityAtom::Invalid.
 CapabilityName findCapabilityName(UnownedStringSlice const& name);
 
+    /// Check if 'name' is an '_Internal' or 'External' capability.
+bool isInternalCapabilityName(CapabilityName name);
```

Comment:
I am not sure if we want to call it an "internal".
How about "terminal" to indicate that the capability doesn't include the lower versions of it.

Reply:
From what I understand the point of having a distinction of '_Internal' versus `External' is that we want the ability to have certain capabilities we can safely change/remove/reformat for the ease of making stdlib capabilities without introducing breaking changes into user code.

These naturally will tend to be a "terminal" like atom but I do not know if this will always be the case.

Reply:
I am ok with internal and external naming here.